draft-ietf-taps-interface.md

```diff
@@ -2388,6 +2408,33 @@ The Capacity Profile for a selected protocol stack may be modified on a
 per-Message basis using the Transmission Profile Message Property; see
 {{send-profile}}.
 
+### Ojective of Using Multiple Paths {#multipath-objective}
```

Comment:
Typo on Ojective -> Objective. Beyond that, I'd call this "Multipath Policy" or something similar. Objective doesn't seem like the right word.

Reply:
This had been my first idea too, but calling it policy does not really tell you whether it is the policy with regards to allowing multipath or the policy how to perform multipath. Objective makes that clear. Any better suggejstions?

Reply:
Sorry Phil, I don't understand "objective" in this context, if we need a word to describe the mode, we need another word.

Reply:
yes objective doesn't seem right. I think policy is fine here.